README.md

```diff
@@ -169,9 +178,10 @@ This plugin creates device plugin endpoints based on the configurations given in
 
 
 |     Field      | Required |        Description        |                       Type - Accepted values                        |                          Example/Accepted values                                                   |
-|----------------|----------|---------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------|
+|----------------|----------|---------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------.........................|
```

Comment:
This formatting breaks the table structure when you view the README.md.

I've updated the table with new contents below. Can you please fix this(copy & paste)?

```
|     Field      | Required |        Description        |                      Type - Accepted values                       |                                      Example/Accepted values                                       |
|----------------|----------|---------------------------|-------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
| "resourceName" | Yes      | Endpoint resource name    | string - must be unique and should not contain special characters | "sriov_net_A"                                                                                      |
| "selectors"    | No       | A map of device selectors | Each selector is a map of string list.                            | "vendors": ["8086"], "devices": ["154c", "10ed"], "drivers": ["vfio-pci"], "pfNames": ["enp2s2f0"] |
| "isRdma"       | No       | Mount RDMA resources      | `bool` - boolean value true or false                              | "isRdma": true                                                                                     |
```